Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml

```diff
@@ -59,7 +63,7 @@
     stealGroup: ClothingShoesBootsMagAdv
 
 - type: entity
-  parent: ClothingShoesBootsMag
+  parent: [ClothingShoesBootsMagBase, BaseScienceContraband]
```

Comment:
these shouldn't be marked as contraband at all. they're made by science for the entire crew

Reply:
I have corrected the science mag boots to have no contraband setting
![image](https://github.com/user-attachments/assets/205ee355-0bf2-4038-941c-f4e3078fd240)